install/upgrade/update_to_dev.php

```diff
@@ -119,15 +158,16 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
    $table = 'glpi_plugin_flyvemdm_packages';
    $migration->addField($table, 'parse_status', "enum('pending', 'parsed', 'failed')",
       ['after' => 'dl_filename', 'default' => 'pending']);
+   $migration->addfield($table, 'plugin_orion_tasks_id', 'integer', ['after' => 'dl_filename']);
```

Comment:
Will it be necessary to add a key for this field?

Reply:
update_to_2_0.php is deleted, its conteint is simply moved in update_to_dev.php, and I grouped the upgrade code per table. Many calls to addKey() were mostly duplicated between each file.